h2o-r/h2o-package/R/stackedensemble.R

```diff
@@ -112,6 +114,7 @@ h2o.stackedEnsemble <- function(x,
                                 metalearner_fold_assignment = c("AUTO", "Random", "Modulo", "Stratified"),
                                 metalearner_fold_column = NULL,
                                 metalearner_params = NULL,
+                                metalearner_transform = c("NONE", "Logit", "PercentileRank"),
```

Comment:
Maybe we should use:  `c("none", "logit", "percentile_rank" )` instead?  That way it can match Python too.

Reply:
This is generated from the schema - I don't know if I can influence this.

Reply:
@tomasfryda There are many sets of arg values in Python/R which differ between what's in the schema (we could also change the schema) and what we have in Python/R.  In the past few years, there's been a push in H2O to standardizing arg names on what's good for Python/R (which is lower/snake_case).  https://github.com/h2oai/h2o-3/blob/59b4e6fc81912dc3b067f50346aaa54f2c52faf1/h2o-algos/src/main/java/hex/schemas/StackedEnsembleV99.java#L106

The Python names are the snake case version, so right now it's unnecessarily inconsistent between Python and R.  Are you saying we can update the arg names in Python (after the client API files are generated from the schema), but not in R?: https://github.com/h2oai/h2o-3/blob/59b4e6fc81912dc3b067f50346aaa54f2c52faf1/h2o-py/h2o/estimators/stackedensemble.py#L561

I wanted to see if there's an existing convention for "None" but we have all three versions represented in various parts of the H2O algorithms: `"NONE", "none", "None"`. :-/

Reply:
```
#' @param metalearner_fold_assignment Cross-validation fold assignment scheme for metalearner cross-validation.  Defaults to AUTO (which is
#'        currently set to Random). The 'Stratified' option will stratify the folds based on the response variable, for
#'        classification problems. Must be one of: "AUTO", "Random", "Modulo", "Stratified".
```

just few lines above, seems consistent with what we have